accord-core/src/main/java/accord/utils/Timestamped.java

```diff
@@ -17,4 +17,10 @@ public static <T> Timestamped<T> merge(Timestamped<T> a, Timestamped<T> b)
     {
         return a.timestamp.compareTo(b.timestamp) >= 0 ? a : b;
     }
+
+    @Override
+    public String toString()
```

Comment:
given that this also has timestamp would it make sense to add to string as well?

Reply:
Maybe, though I think when debugging it's important to balance information, and it's easier for humans to parse maps with just the value string IME. It's rare the timestamp is important, so I preferred to prioritise the more common case.